### DISCUSSION TITLE
crawlWrap - crwData return

### DISCUSSION THREAD

**Discussion by dantzert** [bmcclintock/momentuHMM#67] [Q&A] [answered]
On the [example data](https://raw.github.com/bmcclintock/momentuHMM/master/vignettes/nfsData.RData) from the northern fur seal script the following line executes without issue:

crwOut_nfs <- crawlWrap(nfsData, timeStep = "4 hours")

I've processed another data source to be in a format that seems identical to the above. Both objects consist of 7 columns named "ID","time","x","y","ln.sd.x","ln.sd.y","error.corr"  (in that order) with the same datatypes in each column. 

On execution of this line:

crwOut_fox <- crawlWrap(crawl_data, timeStep = "4 hours")

I receive the following error:

Error in crwData(list(crwFits = model_fits, crwPredict = predData)) : 
  Can't construct crwData object: fields are missing

which seems to refer to line 626 in [crawlWrap.R](https://github.com/bmcclintock/momentuHMM/blob/master/R/crawlWrap.R)

I thought the issue might be the definition of predData within the HeirInd() control block at line 584 but this wouldn't explain why the NFS code executes. What could be causing this error?

The command I used to retrieve the seal data
load(url("https://raw.github.com/bmcclintock/momentuHMM/master/vignettes/nfsData.RData"))
also returns foragedives and the "Par" list. Should either of these be relevant?

[fox_crawl.xlsx](https://github.com/bmcclintock/momentuHMM/files/4881029/fox_crawl.xlsx)

**Comment by bmcclintock** [comment, marked as answer]
I need a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_working_example#:~:text=In%20computing%2C%20a%20minimal%20working,to%20be%20demonstrated%20and%20reproduced.&text=A%20minimal%20working%20example%20may,short%20self%2Dcontained%20correct%20example.). If you can provide one, I can try and diagnose.

**Comment by dantzert** [comment]
Had some malformed input files. Sorry for the bother
